Src/CSharpier.CLI/CSharpier.CLI.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <Import Project="../../CSharpier.Build.props" />
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <PackageId>CSharpier</PackageId>
+        <AssemblyName>dotnet-csharpier</AssemblyName>
+        <TargetFramework>net5.0</TargetFramework>
+        <PackAsTool>true</PackAsTool>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Ignore" Version="0.1.42" />
+        <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
+        <PackageReference Include="System.CommandLine" Version="2.0.0-beta1.21308.1" />
+        <PackageReference Include="System.IO.Abstractions" Version="13.2.29" />
+        <PackageReference Include="System.IO.Abstractions.TestingHelpers" Version="13.2.29" />
+        <PackageReference Include="System.Text.Encoding.CodePages" Version="5.0.0" />
+        <PackageReference Include="UTF.Unknown" Version="2.3.0" />
+        <PackageReference Include="YamlDotNet" Version="11.1.1" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
+            <_Parameter1>CSharpier.Tests</_Parameter1>
```

Comment:
Woah... we don't do the `AssemblyInfo.cs` anymore? :open_mouth:

Reply:
It is autogenerated now, although I did find a cleaner way to do this, because it is pretty gross.